.github/workflows/test-cpu.yml

```diff
@@ -8,7 +8,7 @@ on:
   pull_request:
 
 env:
-  PYTEST_ADDOPTS: "-v --color=yes"
+  PYTEST_ADDOPTS: "-v --color=yes -n auto --internet-tests --junitxml=test-data/test-results.xml"
```

Comment:
`--internet-tests` doesn’t exist here, it’s a custom flag we define in scanpy’s test config